locustempus/main/management/commands/integrationserver.py

```diff
@@ -56,22 +56,41 @@ def handle(self, *fixture_labels, **options):
             email='superuser@example.com',
             is_superuser=True
         )
-        course: Course = CourseFactory.create()
-        student: User = UserFactory.create(
-            username='student',
+        # Sandbox Course
+        c1: Course = SandboxCourseFactory.create()
```

Comment:
a minor point, but the old names `course`, `student`, `faculty` were much clearer. no need to change.